WordPress/src/main/res/drawable/people_list_divider.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<inset xmlns:android="http://schemas.android.com/apk/res/android"
+       android:insetLeft="@dimen/people_list_divider_left_margin">
```

Comment:
Hmm, this seems to adhere to the Material Design specs but we don't put such a margin in the rest of the lists in the app (where we use full width separators). This difference make this one look kinda strange to my eyes.

Can we perhaps defer this change to a future release, where we will change all suitable separators to have the margin? /cc @rickybanister